<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Phonebook</class>
 <widget class="QMainWindow" name="Phonebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phonebook</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="searchLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Search Contacts</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="phoneLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addContactButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Contact</string>
    </property>
   </widget>
   <widget class="QLabel" name="appNameLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Contact Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showAllContactsButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>520</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>List Contacts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>520</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
   </widget>
   <widget class="QListWidget" name="contactListWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>331</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearListButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearScreen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;
    &lt;body&gt;
        &lt;p&gt;&amp;copy; 2024 Team 07 SEA:ME&lt;/p&gt;
    &lt;/body&gt;
&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
